drivers/storage/portworx/deployment.go

```diff
@@ -510,26 +573,32 @@ func (t *template) getArguments() []string {
 		}
 
 	} else if t.cluster.Spec.CloudStorage != nil {
-		if t.cluster.Spec.CloudStorage.DeviceSpecs != nil {
+		if t.cloudConfig != nil && len(t.cloudConfig.CloudStorage) > 0 {
+			// CapacitySpecs have higher preference over DeviceSpecs
+			for _, cloudNodeSpec := range t.cloudConfig.CloudStorage {
```

Comment:
nit: Can we call this cloudDriveSpec? Node spec is a bit misleading.

Reply:
done!